src/test/java/com/thirdcc/webapp/web/rest/UserResourceIT.java

```diff
@@ -710,6 +720,78 @@ public void getAllNotEventCrewUsers_WithEventCrewUser() throws Exception  {
             .andExpect(jsonPath("$.[*].imageUrl").value(not(hasItem(DEFAULT_IMAGEURL))));
     }
 
+    @Test
+    @Transactional
+    public void getUserWithoutFamily_ShouldReturn200() throws Exception {
+        userRepository.deleteAll();
+       userRepository.saveAndFlush(user);
+
+        // Get all the users
+        restUserMockMvc.perform(get("/api/users?family=false")
+            .accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk())
+            .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
+            .andExpect(jsonPath("$.[*].firstName").value(hasItem(user.getFirstName())))
+            .andExpect(jsonPath("$.[*].lastName").value(hasItem(user.getLastName())))
+            .andExpect(jsonPath("$.[*].email").value(hasItem(DEFAULT_EMAIL)))
+            .andExpect(jsonPath("$.[*].imageUrl").value(hasItem(DEFAULT_IMAGEURL)))
+            .andExpect(jsonPath("$.[*].langKey").value(hasItem(DEFAULT_LANGKEY)));
+    }
+
+    @Test
+    @Transactional
+    public void getUserWithoutFamily_ShouldReturnEmptyList() throws Exception {
+        User savedUser = userRepository.saveAndFlush(user);
+        int databaseSizeBeforeUpdate = userRepository.findAll().size();
+
+        UserCCInfo userCCInfo = createUserCCInfoEntity();
+        userCCInfo.setUserId(savedUser.getId());
+        userCCInfoRepository.saveAndFlush(userCCInfo);
+
+        // Get all the users
+        restUserMockMvc.perform(get("/api/users?family=false")
+            .accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$", hasSize(databaseSizeBeforeUpdate-1)));
+    }
+
+    @Test
+    @Transactional
+    public void getUserWithFamily_ShouldReturn200() throws Exception {
+        userRepository.deleteAll();
```

Comment:
No need delete again since you have did in @AfterEach right?

Reply:
Resolved in 5940d2bd46afc1053fdcb4d95e313b57bdf44cde